src/components/send/SendContactList.js

```diff
@@ -211,11 +214,17 @@ export default function SendContactList({
     isDarkMode,
   ]);
 
+  const flyInKey = useDeepCompareMemo(() => String(Date.now()), [sections]);
```

Comment:
You can render lots of elements within the same millisecond, so maybe it makes sense to use another key for that

Reply:
But it's a key for the whole list component, does that make sense to leave it as is since it will also change only when sections change. I think they will change when the input changes so it is highly unlikely it will happen on the same millisecond, WDYT?